### PR TITLE
Feature/seo meta descriptions

### PR DESCRIPTION
**Pull request by avi312singh**
wip meta descriptions

### CODE REVIEW

File: src/templates/blog-post.jsx [hunk 1]
```diff
@@ -19,6 +19,13 @@ query BlogPostBySlug($slug: String!) {
   }
   contentfulBlogPost(slug: {eq: $slug}) {
     title
+    pageSeoTitle
```
Comment (eugene-matvejev) [inline src/templates/blog-post.jsx]: why we call pageSeoTitle like this, not just a title?

File: src/templates/page-template.jsx [hunk 2]
```diff
@@ -143,20 +151,37 @@ query componentsForHomePage($slug: String!) {
...
+const Page = ({ data }) => {
+    const page = get(data, 'contentfulPageTemplate');
+    return (
+        <main>
+            <Helmet>
+                <title>{page.pageSeoTitle}</title>
+                <meta name="description" content={page.metaDescription} />
+                {page.metaTags && page.metaTags.map(
+                    ({ name, content }, i) => (
+                        <meta key={i} name={name} content={content.content} />
```
Comment (eugene-matvejev) [inline src/templates/page-template.jsx]: content seems to be a scalar value, so you don't have to do content.content

File: src/templates/page-template.jsx [hunk 3]
```diff
@@ -143,20 +151,37 @@ query componentsForHomePage($slug: String!) {
...
+            <Footer />
+        </main>
+    );
+};
 
 Page.propTypes = {
     data: PropTypes.shape({
         contentfulPageTemplate: PropTypes.shape({
+            pageSeoTitle: PropTypes.string.isRequired,
+            metaTags: PropTypes.shape({
```
Comment (eugene-matvejev) [inline src/templates/page-template.jsx]: this seems to be an array

File: src/templates/blog-post.jsx (hunk 1 above)
Comment (avi312singh) [reply src/templates/blog-post.jsx]: title is the name of the content in contentful.. page seo title is the seo title applied to the page

Comment (eugene-matvejev) [reply src/templates/blog-post.jsx]: but should we call it like this? why not just a title?

File: src/templates/page-template.jsx (hunk 3 above)
Comment (eugene-matvejev) [reply src/templates/page-template.jsx]: Shape !== array

File: src/templates/blog-post.jsx (hunk 1 above)
Comment (avi312singh) [reply src/templates/blog-post.jsx]: yessss :)

File: src/templates/blog-post.jsx [hunk 4]
```diff
@@ -189,17 +190,17 @@ query BlogPostBySlug($slug: String!) {
...
-    const title = get(data, 'site.siteMetadata.title');
     const post = get(data, 'contentfulBlogPost');
     const heroImage = get(data, 'contentfulBlogPost.heroImage');
     const components = get(data, 'contentfulBlogPost.blogContent');
 
     return (
         <main>
-            <Helmet title={`${post.title} | ${title}`}>
+            <Helmet title={post.pageSeoTitle}>
+                <meta name="description" content={post.metaDescription} />
```
Comment (anasufana) [inline src/templates/blog-post.jsx]: Cool 2 in 1!  👍

File: src/templates/__snapshots__/blog-post.test.jsx.snap [hunk 5]
```diff
@@ -5,10 +5,18 @@ exports[`<BlogPost/> render with default/required props 1`] = `
   <HelmetWrapper
     defer={true}
     encodeSpecialCharacters={true}
-    title="{{data.contentfulBlogPost.title}} | {{site.siteMetadata.title}}"
```
Comment (andgreendigital) [inline src/templates/__snapshots__/blog-post.test.jsx.snap]: Do we want this to be hardcoded?

File: src/templates/blog-post.jsx (hunk 1 above)
Comment (andgreendigital) [reply src/templates/blog-post.jsx]: pageSeoTitle is much clearer! Explains exactly what is is and what it is for! 🙂

Comment (andgreendigital) [reply src/templates/blog-post.jsx]: Also to clarify marketing want to have full control of SEO so would want to specifically set SEO titles which may be different to the title of a post.

**Comment by andgreendigital** [comment]
Update PR and resolve any comments above that are unresolved and I will review again! 🙂

**Pull request merged**